### ISSUE TITLE
MacOS Failed to rebuilde native modules : Error: ENOENT: no such file or directory, rename ...odbc_bindings_mac_18.node -> odbc_bindings.node

### ISSUE THREAD

**Issue by BjarniThor** [ibmdb/vscode-extension#47]
Hi
I am using DB2 Connect on MacOS. Have to following version of VSCode.

Version: 1.69.1
Commit: b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a
Date: 2022-07-12T08:21:51.333Z
**Electron: 18.3.5**
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Darwin x64 21.5.0
- Platform:
- vscode Version:
- Is node.js and npm installed: yes/no
- node.js and npm Version:
- Take a screen shot of VScode(Help->About) information and paste it:

Steps to Reproduce:

1. VSCode updated to newest version
2. Install DB2Connect newest version
3. Rebuild native drivers

Fails with following error:
`Error: ENOENT: no such file or directory, rename '/Users/xyz/.vscode/extensions/ibm.db2connect-2.3.0/node_modules/ibm_db/build/Release/odbc_bindings_mac_18.node' -> '/Users/xyz/.vscode/extensions/ibm.db2connect-2.3.0/node_modules/ibm_db/build/Release/odbc_bindings.node'`

Has been working flawlessly for a few months now (using version 2.2.0). Then this starts this morning, pretty sure last time I used it was this monday but may have been before the weekend.

Looking forward to hear ideas from you.

**Comment by bimalkjha** [comment]
@BjarniThor It is due to update of vscode on latest version which uses electron 18. Try below commands:
```
cd /Users/xyz/.vscode/extensions/ibm.db2connect-2.3.0/node_modules/ibm_db/
npm install
```
Now verify. Share the output of `npm install` if issue still unresolved. Thanks.

**Comment by bimalkjha** [comment]
@BjarniThor Added support for electron 18 on mac. Just rebuild the native module and verify.
If issue still persist, uninstall the db2connect extension and reinstall. Please update the result here. Thanks.

**Comment by BjarniThor** [comment]
Sorry for late reply, went on unexpected holidays. 
Problem although still persists, although a bit different. Keep this issue closed, mine is the same as [Issue #50 ](https://github.com/ibmdb/vscode-extension/issues/50)
